aead/src/lib.rs

```diff
@@ -55,154 +73,77 @@ pub trait AeadLeader {
     /// * `explicit_nonce` - The explicit nonce to use for encryption.
     /// * `plaintext` - The plaintext to encrypt.
     /// * `aad` - Optional additional authenticated data.
```

Comment:
since aad is not an `Option<>`, we could remove the word optional from the comments here and below.